check.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -ex
+
+cd $(dirname `realpath $0`)
```

Comment:
```suggestion
cd "$(dirname "$(realpath "$0")")"
```
To make this safe to paths with spaces in them. I highly recommend running [shellcheck](https://github.com/koalaman/shellcheck) on these scripts to catch any other latent errors.